src/components/Navbar.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/button-has-type */
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
```

Comment:
- [x] Linters are tools that perform static analysis on software to detect and report adherence to a set of coding best practices. Instead of disabling eslint, it would be preferable to resolve the issue.

Kindly add `type="submit"` to the button element to fix this.